@here/harp-mapview/lib/ThemeLoader.ts

```diff
@@ -102,6 +102,8 @@ export class ThemeLoader {
             }
             theme = (await response.json()) as Theme;
             theme.url = themeUrl;
+            // Temporary fix until HARP-7424 is fixed.
+            theme = this.resolveUrls(theme);
```

Comment:
@zbigg , is this an acceptable short term solution?

Reply:
At first look should be ok. But who knows all the consequences.

Reply:
@nzjony Fixed (properly) i belive in #866